exercises/nucleotide-codons/description.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-Write a function that returns the name of an amino acid a particular codon, possibly using shorthand, encodes for.
+Translate a given codon into the name of the amino acid that it encodes, possibly using shorthand.
```

Comment:
If we're rewording this, the exercise asks you to translate a sequence of codons into a sequence of amino acids. There's no shorthands in any of the tracks that I've seen.

Reply:
I think that's explained in the description. DNA has the 4 bases, Adenine, Cytine, Thymine, Guanine. The shorthand it talks about is to do with the fact that multiple codons can map to the same amino acid, e.g. AAA and AAG both map to Lysine. So the shorthand they came up with is AAR (R means A|G). I think the "may use shorthand" language is due to the fact that the input might be "AAA", "AAG", or "AAR" and the expected output would be "Lysine"? But I haven't done the exercise yet.

Reply:
Sorry, wrote that from phone while horizontal and in pain 😅. Had a quick chance to search, and -- because the exercise is deprecated? -- I've only found it (locked) on the Rust track. Because of that and the fact that there's no canonical-data.json file I can't confirm my assumption above is correct. 

Since the exercise is deprecated, is it worth further effort to refine the text?

Reply:
> Since the exercise is deprecated, is it worth further effort to refine the text?

Not really